src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx

```diff
@@ -7,6 +7,49 @@ topics:
 
 Use this document to learn how to review and peer edit docs for you fellow writers in GitHub. Check the [Tech Writer workflow doc](/docs/style-guide/writer-workflow/tech-writer-workflow/) for info on how to set up your local environment. To troubleshoot GitHub issues, see [our guide](/docs/style-guide/writer-workflow/github-troubleshooting/).
 
+## What needs a peer review [#peer-reqs]
+
+Any substantial pull request should be peer reviewed. The line between substantial and trivial is a bit blurry, so refer to the following table when in doubt:
+
+<table>
+<thead>
+<tr>
+<th style={{ width: "200px" }}>
+Doesn't need a peer review
+</th>
+<th>
+Needs a peer review
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+Edits less than 5 lines.
+</td>
+<td>
+Edits more than 5 lines.
```

Comment:
I disagree with 5 lines. I think more than 10-15 lines would need a PR.

Reply:
I also wonder if we need some guidelines around number of pages that an edit touched? For example, Both zulu and I apply the exact same text to multiple docs from time to time. The actual text edit may be minimal, but the number of docs may be a lot. I honestly don't think a one line changed applied to 25 docs does not need a peer review, but am interested in other writers' thoughts.

Reply:
I say go with your gut. I (almost) always check stuff locally, that's why I'm confident of the changes.

Reply:
I always check stuff locally or in the cloud preview build. This is super important, I think.

Reply:
@barbnewrelic What if that one line applied to 25 docs could be slightly improved by a second brain thinking about it? Then you've improved 25 docs :) 

There are extremely minor edits that I've made that have been improved by another writer taking a couple minutes to think about it.

Reply:
Yeah, I agree with @paperclypse here. Another point is, if it's just a small change, does it really matter if it takes an extra day or two for someone to review it? 

I do think we can be really flexible with where the line is drawn here though.

Reply:
And for the record, I think this 5 line limit is more so for reviewing the content itself than a fear that a change more than 5 lines could break something.